hnn_core/feed.py

```diff
@@ -127,15 +144,15 @@ def __create_extpois(self):
         #     print(lamtha, np.mean(xdiff), np.var(xdiff), 1/lamtha**2)
         # Convert array into nrn vector
         # if len(val_pois)>0: print('val_pois:',val_pois)
-        self.eventvec.from_python(val_pois)
-        return self.eventvec.size() > 0
+        self.nrn_EventVec.from_python(val_pois)
```

Comment:
variable names should be in snake case unless they are classes!

Reply:
I interpret this as variables names should NOT be in snake unless ...